src/WebView.cpp

```diff
@@ -130,3 +130,21 @@ void WebView::refresh()
 {
     webkit_web_view_reload(*this);
 }
+
+void WebView::zoomIn()
+{
+    double zoomLevel { webkit_web_view_get_zoom_level(*this) };
+    if (zoomLevel >= 2) return;
+
+    zoomLevel += 0.05;
+    webkit_web_view_set_zoom_level(*this, zoomLevel);
+}
+
+void WebView::zoomOut()
+{
+    double zoomLevel { webkit_web_view_get_zoom_level(*this) };
+    if (zoomLevel <= 0.5) return;
```

Comment:
Please use brackets on the next line for convention consistency.

Reply:
Done